hazelcast/src/main/java/com/hazelcast/monitor/LocalPublisherStats.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright (c) 2008-2015, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.hazelcast.monitor;
+
+import com.hazelcast.internal.management.JsonSerializable;
+
+public interface LocalPublisherStats extends JsonSerializable {
```

Comment:
Can you rename it to LocalWanPublisherStats or something? Publisher is such a generic term.

Also this interface needs to be documented since it is public API; the same goes for the other changes you make in the com.hazelcast.monitor package.